web/views.py

```
@@ -10,7 +10,7 @@
 import subprocess
 import time
 from collections import Counter, defaultdict
-from datetime import timedelta
+from datetime import timedelta, datetime
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)